Telerik.JustMock/Helpers/FluentHelper.cs

```diff
@@ -180,6 +177,35 @@ public static void Assert<T>(this T obj, Expression<Action<T>> expression, Occur
 			});
 		}
 
+		/// <summary>
+		/// Asserts the specific call
+		/// </summary>
+		/// <typeparam name="T">Type of the mock.</typeparam>
+		/// <param name="obj">Target mock object</param>
+		/// <param name="action">Propert set action</param>
+		public static void AssertSet<T>(this T obj, Action<T> action, string message = null)
```

Comment:
Could you add a description of the message parameter similar to how the other parameters are described?